File: 프로그래머스/lv0/120924. 다음에 올 숫자/다음에 올 숫자.py
def solution(common):
    answer = 0
    a1,a2,a3 = common[:3] #마지막 3개 가져오기 
    if (a2 - a1) == (a3 - a2):
        answer = common[-1] + (a3 - a2)
    elif (a2 // a1) == (a3 // a2):
        answer = common[-1] * (a3 // a2)
    return answer

File: 프로그래머스/0/120924. 다음에 올 숫자/다음에 올 숫자.java
class Solution {
    public int solution(int[] common) {
        int length = common.length;
        
        if (common[1] - common[0] == common[2] - common[1]) {
            return common[length - 1] + (common[1] - common[0]);
        } else {
            return common[length - 1] * (common[1] / common[0]);
        }
    }
}